apps/next-docs/content/components/Footnotes/react.mdx

```diff
@@ -0,0 +1,152 @@
+---
+
+import {Label} from '@primer/react'
+
+import {FootnotesAsProp} from './react'
+
+```js
+import {Footnotes} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Footnotes>
+  <Footnotes.Item>
+    "Automating your workflow with GitHub Actions," GitHub Docs{' '}
+    <InlineLink href="https://docs.github.com/en/actions">https://docs.github.com/en/actions</InlineLink>.
+  </Footnotes.Item>
+  <Footnotes.Item>
+    "About Issues," GitHub Docs{' '}
+    <InlineLink href="https://docs.github.com/en/issues">https://docs.github.com/en/issues</InlineLink>.
+  </Footnotes.Item>
+  <Footnotes.Item>
+    "GitHub Copilot: Your AI pair programmer," GitHub{' '}
+    <InlineLink href="https://github.com/features/copilot/">https://github.com/features/copilot/</InlineLink>.
+  </Footnotes.Item>
+  <Footnotes.Item>
+    "Explore the GitHub Marketplace," GitHub Docs{' '}
+    <InlineLink href="https://github.com/marketplace">https://github.com/marketplace</InlineLink>.
+  </Footnotes.Item>
+  <Footnotes.Item>
+    "GitHub Pages basics," GitHub Docs{' '}
+    <InlineLink href="https://docs.github.com/en/pages">https://docs.github.com/en/pages</InlineLink>.
+  </Footnotes.Item>
+</Footnotes>
+```
+
+### Paragraphs
+
+```jsx live
+<Footnotes as="div">
+  <Footnotes.Item>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean non facilisis tortor. Pellentesque rhoncus blandit
+    posuere. Sed eget mi finibus, fermentum turpis vitae, eleifend dolor. Maecenas viverra neque at cursus faucibus. Nam
+    tempus enim quis augue pulvinar finibus. Integer accumsan turpis eget mauris pretium, non dapibus orci cursus. Ut in
+    sollicitudin dui, eu tincidunt nunc. Donec ut placerat ex. In ullamcorper nibh eu sapien convallis bibendum. Sed
+    diam leo, rhoncus non viverra nec, ullamcorper quis arcu.
+  </Footnotes.Item>
+  <Footnotes.Item>
+    Etiam scelerisque turpis at massa pellentesque, ut pretium lorem iaculis. Pellentesque feugiat nisi quis viverra
+    posuere. Integer eu pulvinar ligula, lacinia sollicitudin libero. Etiam in cursus lacus, vitae vestibulum elit.
+    Fusce quis purus a urna placerat dignissim non id lectus. Suspendisse malesuada nisi eget nibh tempus, et
+    sollicitudin tortor posuere. Fusce id consectetur purus, sit amet faucibus orci. Lorem ipsum dolor sit amet,
+    consectetur adipiscing elit. Phasellus sollicitudin nisl vel felis tempor, quis porttitor diam elementum. Integer
+    vel enim eget urna ullamcorper euismod eget quis nisl. Pellentesque nec erat tincidunt, dignissim ligula ut,
+    consectetur lectus. Sed scelerisque urna maximus varius mollis. Ut dictum sed nibh id fermentum. Ut hendrerit
+    accumsan tortor, eu tincidunt mauris blandit in. Suspendisse ut velit in felis molestie euismod. Nam vitae dui a
+    justo lacinia semper.
+  </Footnotes.Item>
+</Footnotes>
+```
+
+### Citations with return links
+
+```jsx live
+<>
```

Comment:
FYI this example is not rendering correctly @danielguillan. Might need a stack.

<img width="996" alt="Screenshot 2025-04-23 at 12 05 53" src="https://github.com/user-attachments/assets/1b4640ae-5eb7-4fb1-80ba-0f63b16368d6" />